src/lib/gssapi/krb5/set_context_flags.c

```diff
@@ -0,0 +1,41 @@
+/* -*- mode: c; c-basic-offset: 4; indent-tabs-mode: nil -*- */
+
+#include "k5-int.h"
+#include "gssapiP_krb5.h"
+
+#include <stdlib.h>
+
+OM_uint32 KRB5_CALLCONV
+krb5_gss_set_context_flags(OM_uint32 *minor_status, gss_ctx_id_t context,
+                           uint64_t req_flags, uint64_t ret_flags)
```

Comment:
The I-D says this function gets just three arguments: the minor_status, the context, and `req_flags`.  There should be no way to set `ret_flags`!